src/utils.rs

```diff
@@ -18,12 +18,21 @@ pub fn read_file<P: AsRef<Path> + Debug>(file_name: P) -> Result<String> {
     if result.is_err() {
         log::debug!("Error reading file: {:?}", result);
     } else {
-        log::trace!("File read sucessfully");
+        log::trace!("File read successfully");
     };
 
     result
 }
 
+// Reads command output from stderr or stdout depending on to which stream program streamed it's output
```

Comment:
I'd suggest using triple-slashes here so that the comment is interpreted as documentation.
This ensures that Intellisense and IDEs show the description as part of code-completion 😊
 
```suggestion
/// Reads command output from stderr or stdout depending on to which stream program streamed it's output
```

Reply:
Good to know. Thank you 🙂